packages/mask/src/social-network-adaptor/facebook.com/injection/SearchResultTabs.tsx

```diff
@@ -1,11 +1,11 @@
 import { MutationObserverWatcher } from '@dimensiondev/holoflows-kit'
-import { SearchResultBox } from '../../../components/InjectedComponents/SearchResultBox'
+import { SearchResultTabs } from '../../../components/InjectedComponents/SearchResultTabs'
 import { createReactRootShadowed } from '../../../utils/shadow-root/renderInShadowRoot'
 import { startWatch } from '../../../utils/watcher'
 import { searchResultHeadingSelector } from '../utils/selector'
 
-export function injectSearchResultBoxAtMinds(signal: AbortSignal) {
+export function injectSearchResultTabsAtFacebook(signal: AbortSignal) {
```

Comment:
```suggestion
export function injectSearchResultTabsAtMinds(signal: AbortSignal) {
```

Or just emit the SNS name since we are already in the context of the folder.

```suggestion
export function injectSearchResultTabs(signal: AbortSignal) {
```

Reply:
It's an issue from GitHub differ.

Reply:
I see. You rename the file.